imports/plugins/core/payments/server/no-meteor/schemas/schema.graphql

```diff
@@ -55,14 +55,14 @@ type ExampleIOUPaymentData {
   fullName: String!
 }
 
-# These should be defined in their respective plugin schemas, but `extend union` isn't working yet
-union PaymentData = ExampleIOUPaymentData | StripeCardPaymentData | MarketplaceStripeCardPaymentData
+# Use `extend union` to define a payment plugin's PaymentData type
+"Data identifying a payment for an order"
+union PaymentData = ExampleIOUPaymentData
 
-# These should be defined in their respective plugin schemas, but `extend enum` isn't working yet
+# Use `extend enum` to add a plugin's payment method name
 "The name of a payment method, which is how payment methods are keyed"
 enum PaymentMethodName {
   iou_example
-  stripe_card
```

Comment:
Why do we remove stripe here without extending the enum elsewhere?

Reply:
The extension has always been in the stripe plugin here:

https://github.com/reactioncommerce/reaction/blob/f9fd923a4a752e9a996853094693d7d48f2d05d7/imports/plugins/included/payments-stripe/server/no-meteor/schemas/schema.graphql#L43-L45

It just did not work before.